docs/user-guide/solver/04-parameters.md

```diff
@@ -472,13 +472,17 @@ _**This section is under construction**_
 
 ---
 #### solver-parameters
-[//]: # (TODO: document this parameter)
-_**This section is under construction**_  
 
-- **Expected value:**
-- **Required:** **yes**
-- **Default value:**
-- **Usage:**
+- **Expected value:** Any string
+- **Required:** no
+- **Default value:** `""`
+- **Usage:** This parameter has effect only if:
+  - or-tools is used (argument `--use-ortools` from the command line) 
+  - solver `xpress` or `scip` is selected (using `--ortools-solver=<solver>` from the command line) as `sirius, glpk, coin` do not implment this functionality in or-tools. 
+  
+  The format is solver-specific and is the same as the corresponding solver configuration file format. This is an advanced usage and the user should therefore refer to the specific solver documentation to know how to specify the parameters. We give an example to set the maximum number of threads used during solve to 1:
+  - With Xpress : `solver-parameters = THREADS 1`
+  - With SCIP : `solver-parameters = parallel/maxnthreads = 1`
```

Comment:
can you add a second parameter in these examples please? (for example MAXNODE for xpress => `THREADS 1 MAXNODE 300000`, and its equivalent in SCIP, or any other parameter)
to allow the users to see the syntax to use to define multiple parameters in this single string (eq in xpress parameters are sperated by an empty space)